File: SingleAPIExample/CSharp/Get.cs
using Splitit.SDK.Client.Api;
using Splitit.SDK.Client.Model;

namespace Examples
{
    public class GetExample
    {
        InstallmentPlanApi PlanApi { get; }

        public void Get(string installmentPlanNumber)
        {
            var request = new GetInstallmentsPlanSearchCriteriaRequest
            {
                QueryCriteria = new InstallmentPlanQueryCriteria
                {
                    InstallmentPlanNumber = installmentPlanNumber,
                }
            };
            var getResponse = PlanApi.InstallmentPlanGet(request);
        }
    }
}

File: SingleAPIExample/PHP/Get.php
<?php
$sessionId = '{{SessionID}}';
\SplititSdkClient\Configuration::sandbox()->setApiKey('{{APIKey}}');

$apiInstance = new SplititSdkClient\Api\InstallmentPlanApi(\SplititSdkClient\Configuration::sandbox(), $sessionId);

$request = new \SplititSdkClient\Model\GetInstallmentsPlanSearchCriteriaRequest([
    'query_criteria' => [
        "InstallmentPlanNumber" => "{{InstallmentPlanNumber}}"
    ],
]);

try {
    $result = $apiInstance->installmentPlanGet($request);
    print_r($result);
} catch (Exception $e) {
    echo 'Exception when calling InstallmentPlanApi->installmentPlanGet: ', $e->getMessage(), PHP_EOL;
}
